doc/man1/openssl-ocsp.pod.in

```diff
@@ -171,17 +179,6 @@ the time that the responder is willing to wait for the client request.
 This time is measured from the time the responder accepts the connection until
 the complete request is received.
 
-=item B<-multi> I<process-count>
-
-Run the specified number of OCSP responder child processes, with the parent
-process respawning child processes as needed.
-Child processes will detect changes in the CA index file and automatically
-reload it.
-When running as a responder B<-timeout> option is recommended to limit the time
-each child is willing to wait for the client's OCSP response.
-This option is available on POSIX systems (that support the fork() and other
-required unix system-calls).
-
```

Comment:
This was erroneously listed as a client option. I moved it down to the server options. (TODO: forgot to move it in the SYNOPSIS section.)

Reply:
Done in 4f7848c801e167f7b2481c665333c867c9f4e86d.